lib/fs/basicfs_copy_range.go

```diff
@@ -4,6 +4,8 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
+// +build !ios
```

Comment:
Is this required? I don't see a corresponding thing adding some other code, so this looks like it excludes a couple of functions that are anyway not referred to because the calling code is excluded already.

Reply:
Did not notice these are actually unused, so this directive can indeed be removed